### PR TITLE
Github integration update

### PR DESCRIPTION
**Pull request by l0k0ms**
Upgrading Github integration to make it fit new configuration process.

Preview: https://d2cx6t4ssmwdrv.cloudfront.net/gus/git-integration-update/integrations/github/

### CODE REVIEW

**Comment by l0k0ms** [comment]
If this is merged, then we can remove: 
https://help.datadoghq.com/hc/en-us/articles/115003314766-Why-events-don-t-appear-to-be-showing-up-in-the-event-stream-with-my-github-integration-

File: content/integrations/github.md [hunk 1]
```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
   * Identify when new code changes lead to system alerts or build failures
   * Discuss code changes with your team in the Datadog Event Stream
 
-
+{{< img src="git_integration_screen.png" alt="Git event" >}}
 
 ## Configuration
 
+1. Add this URL to your Github Webhook, **Settings>Webhooks>Add new Webhook** with this URL:
```
Comment (kmshultz) [inline content/integrations/github.md]: "In Github, add this URL to your webhooks (**Settings>Webhooks>Add new Webhook**):"

File: content/integrations/github.md [hunk 2]
```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
...
+```
+https://app.datadoghq.com/intake/webhook/github?api_key=<YOUR_DATADOG_API_KEY>
+```
 
-Select 'Github' [on the account settings screen][1] and link your account. You can then select which repos you would like to integrate, which branches, and if you'd like to receive commits and/or issues.
+2. Select what you want to send to datadog, we support:
```
Comment (kmshultz) [inline content/integrations/github.md]: Select what you want to send to Datadog. We support:

File: content/integrations/github.md [hunk 3]
```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
...
+3. Add the branches you wish to monitor for each repository. Wildcards are also supported to add all repositories for a user or Organization.
 
+## What to Expect
 
 Once the integration is complete, whatever you select (commits and/or issues) will populate into your Datadog Event Stream. If you view a dashboard, in the top left search bar you can type `sources:github` to see github events overlayed over your the graphs on that dashboard.
 
-   [1]: https://app.datadoghq.com/account/settings
+## FAQ
+
+**Why events don't appear to be showing up in my datadog event stream with my github integration?**
```
Comment (kmshultz) [inline content/integrations/github.md]: Why aren't Github events showing up in my Datadog event stream?

File: content/integrations/github.md [hunk 4]
```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
...
+
+If you have setup your webhook on the relevant Github repos and you can see it's sending data but events don't appear to be showing up in the event stream this might come from your webhook settings:
+
+Instead of having your webhook configured with `content-type:application/x-www-form-urlencoded`
+
+You should set your webhook with `content-type:application/json`.
+{{< collapse id="collapse_screen_github" >}}click here {{< /collapse >}} to see a screenshoot.
```
Comment (kmshultz) [inline content/integrations/github.md]: Is the screenshot really that large? I think let's just show it – no expand/collapse.

File: content/integrations/github.md (hunk 3 above)
Comment (l0k0ms) [inline content/integrations/github.md]: 👍

File: content/integrations/github.md (hunk 4 above)
Comment (l0k0ms) [inline content/integrations/github.md]: 👍

File: content/integrations/github.md [hunk 5]
```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
...
 Once the integration is complete, whatever you select (commits and/or issues) will populate into your Datadog Event Stream. If you view a dashboard, in the top left search bar you can type `sources:github` to see github events overlayed over your the graphs on that dashboard.
 
-   [1]: https://app.datadoghq.com/account/settings
+## FAQ
+
+**Why events don't appear to be showing up in my datadog event stream with my github integration?**
+
+If you have setup your webhook on the relevant Github repos and you can see it's sending data but events don't appear to be showing up in the event stream this might come from your webhook settings:
+
+Instead of having your webhook configured with `content-type:application/x-www-form-urlencoded`
```
Comment (kmshultz) [inline content/integrations/github.md]: If your web hook is configured with `content-type:application/x-www-form-urlencoded`, set it to `content-type:application/json` instead.

File: content/integrations/github.md [hunk 6]
```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
...
+## What to Expect
 
 Once the integration is complete, whatever you select (commits and/or issues) will populate into your Datadog Event Stream. If you view a dashboard, in the top left search bar you can type `sources:github` to see github events overlayed over your the graphs on that dashboard.
 
-   [1]: https://app.datadoghq.com/account/settings
+## FAQ
+
+**Why events don't appear to be showing up in my datadog event stream with my github integration?**
+
+If you have setup your webhook on the relevant Github repos and you can see it's sending data but events don't appear to be showing up in the event stream this might come from your webhook settings:
```
Comment (kmshultz) [inline content/integrations/github.md]: This is just a re-wording of the question. Plus, we can assume if the user is reading FAQ, he has already attempted to "setup your web hook on the relevant Github repos".

Let's delete this sentence.

File: content/integrations/github.md [hunk 7]
```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
...
-Select 'Github' [on the account settings screen][1] and link your account. You can then select which repos you would like to integrate, which branches, and if you'd like to receive commits and/or issues.
+2. Select what you want to send to datadog, we support:
+     * Push (new commits), 
+     * Create and delete (for tags), 
+     * Pull requests,
+     * Issues, 
+     * All comments.
 
-## What to Expect
+3. Add the branches you wish to monitor for each repository. Wildcards are also supported to add all repositories for a user or Organization.
```
Comment (kmshultz) [inline content/integrations/github.md]: "For each repository, add the branches you wish to monitor. If you want to add all repositories for a user or organization, use wildcards." (what's an example of using wildcards?)

Comment (l0k0ms) [inline content/integrations/github.md]: Some example are given on the integration tile https://app.datadoghq.com/account/settings#integrations/github

**Pull request merged**